.docker/testing/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# Dockerfile for gvm-libs-$VERSION-$COMPILER-testing
+
+# Define ARG we use through the build
+ARG VERSION=master
+ARG BUILD_TYPE=Debug
+ARG COMPILER=gcc
+
+# Use '-slim' image for reduced image size
+FROM debian:buster-slim
+
+# This will make apt-get install without question
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Redefine ARG we use through the build
+ARG VERSION
+ARG BUILD_TYPE
+ARG COMPILER
+
+WORKDIR /usr/local/src
+
+# Install core dependencies required for building and testing gvm-libs
+RUN apt-get update \
+    && apt-get install --no-install-recommends --assume-yes \
+    ca-certificates \
+    cmake \
+    git \
+    libglib2.0-dev \
+    libgnutls28-dev \
+    libgpgme-dev \
+    libhiredis-dev \
+    libpcap-dev \
+    libssh-gcrypt-dev \
+    libxml2-dev \
+    libnet1-dev \
+    make \
+    pkg-config \
+    uuid-dev \
+    libssl-dev \
+    lcov \
+    libical-dev \
+    libpq-dev \
+    libnet1-dev \
+    postgresql-server-dev-all \
+    xsltproc \
```

Comment:
i am not sure about the dependencies. looks like the dependencies from gvmd to me. i don't think gvm-libs needs postgres at all and also not xsltproc. but maybe i am wrong.